values.yaml

```diff
@@ -623,6 +623,14 @@ st2sensorcontainer:
 # The st2client deployment/pod simplifies ad-hoc administration.
 # st2client is a special purpose actionrunner pod, but you can customize it separately
 st2client:
+  # st2client config (~/.st2/config) template.
+  # You can access env variables here because this is used in a bash heredoc.
```

Comment:
Nice!
Let's just include a link to https://docs.stackstorm.com/reference/cli.html#configuration-file in comments for both Helm value definitions.

That'll help users find the use cases and configuration examples easily.

Reply:
done